src/wwwroot/partials/signup/confirmationPage.html

```diff
@@ -18,24 +18,24 @@
       <div class="flex margin--bottom-inputs">
         <div class="wrapper-input width--full margin--right-inputs">
           <label class="label--regular" for="industry">{{'industry_text' | translate}}:</label>
-          <ui-select ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
+          <ui-select id="confirmation-select-industry"  ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
```

Comment:
It seems that there is a way to generate the input id, see here: https://github.com/angular-ui/ui-select/blob/32829a39e8efe7f94de049c6b529c23fedfc6686/src/uiSelectDirective.js#L34

What do you think to use it to apply the id to the generated input in place of the generated div (or span)?